models/Element/AbstractElement.php

```diff
@@ -867,4 +868,26 @@ protected function renewInheritedProperties()
         $inheritedProperties = $this->getDao()->getProperties(true);
         $this->setProperties(array_merge($inheritedProperties, $myProperties));
     }
+
+    /**
+     * @internal
+     * @param bool|array $params
+     * @return array
+     */
+    protected static function prepareGetByIdParams(/*array */$params, string $method): array
+    {
+        if($params === true) {
+            trigger_deprecation('pimcore/pimcore', '10.5', 'Using $force=true on %s is deprecated, please use array-syntax [force=>true] instead.', $method);
+            $params = ['force' => true];
+        }
```

Comment:
Maybe we should check for `bool` here? Passing `false` is deprecated as well, I suppose.

```suggestion
        if(is_bool($params)) {
            trigger_deprecation('pimcore/pimcore', '10.5', 'Using $force=%s on %s is deprecated, please use array-syntax [force=>true] instead.', $force ? 'true' : 'false', $method);
            $params = ['force' => $force];
        }
```

Reply:
Hmm, the problem is that `false` is the default value, so when calling `getById(123)` without the second argument a deprecation message gets issued.